src/guc.c

```diff
@@ -873,6 +911,50 @@ _guc_init(void)
 							NULL,
 							NULL);
 
+	DefineCustomIntVariable(MAKE_EXTOPTION("continuous_aggregate_processing_wal_batch_size"),
```

Comment:
```suggestion
	DefineCustomIntVariable(MAKE_EXTOPTION("cagg_processing_wal_batch_size"),
```
All Continuous Aggregate related GUCs we use `cagg`:
```sql
fabrizio=# select name, setting from pg_settings where name ~ 'timescaledb' and name ~ 'cagg';
                        name                        | setting 
----------------------------------------------------+---------
 timescaledb.debug_allow_cagg_with_deprecated_funcs | off
 timescaledb.enable_cagg_create                     | on
 timescaledb.enable_cagg_reorder_groupby            | on
 timescaledb.enable_cagg_sort_pushdown              | on
 timescaledb.enable_cagg_watermark_constify         | on
 timescaledb.enable_cagg_window_functions           | off
 timescaledb.enable_merge_on_cagg_refresh           | off
(7 rows)

fabrizio=# select name, setting from pg_settings where name ~ 'timescaledb' and name ~ 'cont';
 name | setting 
------+---------
(0 rows)
```

Reply:
Done.